src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/BookingService.kt

```diff
@@ -194,10 +193,7 @@ class BookingService(
                 name = approvedPremises.name,
                 apCode = approvedPremises.apCode,
                 legacyApCode = approvedPremises.qCode,
-                probationArea = ProbationArea(
-                  code = staffDetails.probationArea.code,
-                  name = staffDetails.probationArea.description
-                )
+                localAuthorityAreaName = approvedPremises.localAuthorityArea!!.name
```

Comment:
The probation area here was intended to be the probation area of the AP itself. But on further investigation I realise that a premises isn't associated with a probation area. It is associated with:

- **local authority area** e.g. Kirklees, Bradford
- **probation region** e.g. Yorkshire & The Humber, North East
- **ap area** e.g. North East (through probation region)

For reporting purposes I think local authority area is the most useful information to include, but will check this with AJB / performance hub. 

This change is going to break [the "integration" which consumes this event](https://github.com/ministryofjustice/hmpps-probation-integration-services/blob/98701178578ed616323657da2b4ebd2979fc4322/projects/approved-premises-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/service/ApprovedPremisesService.kt#L107-L119) as the premises' probation area is being used to set the `team` on the created Delius contact. 

Perhaps the probation integrations team could derive the correct team from the `premises.legacyApCode` instead? That's the Delius "Qcode" e.g. `Q057`. 

☝️ @marcus-bcl @anthony-britton-moj

Reply:
✅